src/cli/index.js

```diff
@@ -0,0 +1,105 @@
+// Local imports
+import chalk from 'chalk';
+
+import {
+  initFiles,
+  updateFile,
+  deleteFile,
+  updateClassNames,
+} from './actions';
+import { requireTemplates, writeStore } from './requireTemplates';
+import { matchKssDir } from './utils';
+import program from './parseArgs';
+import startWebpack from './server';
+import { defaultStore, config } from './defaultStore';
+import gaze from './fileWatcher';
+
+/**
+ * Initialize data store with files from gaze and original data structure
+ *
+ * @global
+ */
+const store = initFiles(gaze.watched(), defaultStore);
+const huron = defaultStore.get('config');
+
+requireTemplates(store);
+writeStore(store);
+
+if (!program.production) {
```

Comment:
How would you feel about refactoring this so it was like 
```js
if (program.production) {
  gaze.close();
  // start webpack
 exit(); // or return or whatever
} 
```
Then do all the gaze stuff.
And instead of having a `newStore` variable that can be set in lots of places, we could just set a const and save that for each gaze listener. I think that will be easier to follow the flow control.

Reply:
Definitely to the first part, the second I think has to remain ugly because we need a single, app-scoped instance of `newStore` that persists with every file change. My plan here is to just wait until there's time to refactor using `redux`.